test/benchmarks/Benchmarks.Trace/Properties/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think you can delete this file. I'll bet VS is adding it automatically but you can add this to the csproj to stop it from doing that:
```xml
    <!-- Tell Visual Studio to not create a new launchSettings.json file, even though we have AspNetCore assets -->
    <NoDefaultLaunchSettingsFile>true</NoDefaultLaunchSettingsFile>
```

Reply:
TIL